<!-- Launches crop image   -->

<launch>
<!-- <param name="/use_sim_time" value="true"/> -->

   <node pkg="multisense" type="crop_image" name="crop_image" output="screen">    
    <param name="object_name" value="shelf" />
    <remap from="/image" to="/head_mount_kinect/rgb/image_rect_color" />
    <remap from="/trigger_message" to="/ap_ms/run_object_detection" />     
    </node>   
  
   
</launch>
